Frameworks/BlockchainCertificates/Requests/CertificateValidationRequest.swift

```diff
@@ -189,9 +190,42 @@ public class CertificateValidationRequest : CommonRequest {
                     return
                 }
                 
-                self.localHash = sha256(data: stringData)
+                self.localHash = hexStringFromData(input: sha256(data: stringData) as NSData)
+
+                self.state = .fetchingRemoteHash
+            })
+        } else {
+            let docData : Data!
+            do {
+                var json = try JSONSerialization.jsonObject(with: certificate.file, options: []) as! [String: Any]
+                json.removeValue(forKey: "signature")
```

Comment:
that is how json-ld signatures work -- it's equivalent to how we were separating the cert into document and signature